ov/firmware_drivers.go

```diff
@@ -149,12 +149,15 @@ func (c *OVClient) GetFirmwareBaselineById(id string) (FirmwareDrivers, error) {
 
 func (c *OVClient) GetFirmwareBaselineByNameandVersion(name string) (FirmwareDrivers, error) {
 	var fwname, version string
-	fwNameVersion := strings.SplitAfter(name, ",")
-	if len(fwNameVersion) < 1 {
+	fwNameVersion := strings.Split(name, ",")
+	if len(fwNameVersion) == 0 {
```

Comment:
do we have unit tests written considering different scenario where user has send empty, only name/only version or both.
If not can we test those scenarios manually now

Reply:
Yes, it is tested.

If no string is provided(len==0) we are returning an error.
Also, if only version is provided( fwname=="") we are returning an error.